REQUIRE

```diff
@@ -4,7 +4,7 @@ Distributions
 Docile
 DualNumbers
 FITSIO
-ForwardDiff 0.0.2 0.0.3
+ForwardDiff 0.0.2 0.0.3+
```

Comment:
Let's do it, package caching would make my life much better.

On Thu, Sep 24, 2015 at 2:51 PM, Jeffrey Regier <notifications@github.com>
wrote:

> In REQUIRE
> <https://github.com/jeff-regier/Celeste.jl/pull/62#discussion_r40377997>:
>
> > @@ -4,7 +4,7 @@ Distributions
> >  Docile
> >  DualNumbers
> >  FITSIO
> > -ForwardDiff 0.0.2 0.0.3
> > +ForwardDiff 0.0.2 0.0.3+
>
> It's stable, they're on release candidate 2 now. I think we just need one
> more change to PriorParams to make Celeste compatible with 0.4
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jeff-regier/Celeste.jl/pull/62/files#r40377997>.
>